src/stylesheets/index.css

```diff
@@ -45,27 +45,3 @@
 .link {
   @apply text-navy-700 hover:underline font-semibold;
 }
-
-.video-iframe {
-  aspect-ratio: 16/9;
-  width: 100%;
-}
-
-/* Fallback for browsers that don't support aspect-ratio */
```

Comment:
I removed this fallback because I figured that the browser support is good enough https://caniuse.com/mdn-css_properties_aspect-ratio
Unless we need to support IE11? But I didn't think that this was necessary.
Please let me know if I'm wrong and I'll revert this change!